src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue

```diff
@@ -24,42 +24,14 @@
 		<div class="main-panel columns small-12 medium-5 small-order-2 medium-order-1">
 			<div class="info-tab-selector show-for-medium">
 				<button
+					v-for="{title, id} in tabs"
```

Comment:
I like that these are dynamic now. You might want to create a computed version of the tab data so you can filter for active/inactive sections based on the loan type.

Reply:
That’s a cool idea!